### DISCUSSION TITLE
Welcome to streamlit-template Discussions!

### DISCUSSION THREAD

**Discussion by giswqs** [giswqs/streamlit-template#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AliceKoh** [comment]
Hi giswqs,
I have problem with my app deployment with error message: ModuleNotFoundError: No module named 'streamlit_option_menu'. My anaconda do have 'streamlit_option_menu' so I am so confused. Please assist.

I was told to add 'streamlit_option_menu' to requirement.txt and I happen to see your template has this requirement.txt file but how do you generate that? Just manually create at Github under the code and save the file name requirement.txt? I done that but not working. still with same error message.

![image](https://user-images.githubusercontent.com/109507439/182753919-389c3007-2e46-43d0-abf1-ebda37a79dcd.png)

![image](https://user-images.githubusercontent.com/109507439/182753975-e3156099-e0b8-42b5-bd27-d9b0903b1751.png)

![image](https://user-images.githubusercontent.com/109507439/182754167-ad2b9dbd-0300-48f8-9f2c-03473a747204.png)


![image](https://user-images.githubusercontent.com/109507439/182754082-699e529c-62ea-40d2-bc34-d3d5d14d9ca2.png)

**Reply by AliceKoh** [reply]
HI giswqs,
I manage to create the requirements.txt and resolve the No module named 'streamlit_option_menu. However it gives me another problem. No Module named numpy. Actually my app needs to import pandas but dont need to import numpy. Without numpy they works. So I am not sure why, streamlit says No module named numpy as I didnt need numpy. Do you happen to know why?

**Reply by jaravad** [reply]
Hi @AliceKoh, how did you created the requirements.txt?, I'm having the same issue with streamlit_option_menu

**Reply by giswqs** [reply]
I am no longer use Streamlit actively. Check out https://github.com/opengeos/solara-template
